javascript/javascript3/week1/weatherApp/myInterest.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "name": "Bikram",
+        "isMarried": true,
+        "hobbies": ["music","coding","sports"],
+        "birthYear": 1990,
+        "kids":null,
+        "visitedCountries": [ {
+            "countryName": "Austria",
+            "visitYear": 2017,
+            "duration": "7 days"
+            },
+            {
+                "countryName": "USA",
+                "visitYear": 2018,
+                "duration": "15 days",
+                "withFamily": false,
+                "accidents":null
+            }
+        ]
+    },
+    {
+        "title": "pets",
+        "numberOfPets": 2,
+        "haveDog": true,
+        "petsName": ["cat","dog"],
+        "extraPets": []
+    }
+]
```

Comment:
Nice job here, I like that you have quite a bit of nested properties in your JSON. If you don't want to, you don't need to wrap it in the array brackets but it's just a minor thing.